<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditProductDialog</class>
 <widget class="QDialog" name="EditProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>190</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Product name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>label</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_edit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Price</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>price_line</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="price_line"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="upload_checkbox">
           <property name="text">
            <string>Upload thumbnail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="choose_image_button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="image_preview">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="change_button">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
